### ISSUE TITLE
Cannot configure integration - Stuck at "Alexa Media Player - Action Required" screen

### ISSUE THREAD

**Issue by mikesalz** [custom-components/alexa_media_player#990]
**Describe the bug**
Integration was previously working for months. This week, it asked me to reconfigure.  Since then, I cannot get past the "Alexa Media Player - Action Required" screen.

**To Reproduce**
1. Uninstall the integration
2. Reinstall the integration
3. Try to configure it
4. Approve the Amazon login attempt 
5. Click Submit on the "Alexa Media Player - Action Required" screen
6. Shout expletives because you've tried this a dozen times and it doesn't work

**Expected behavior**
After Step 5 above, the integration should work

**Screenshots**
Stuck here:

![image](https://user-images.githubusercontent.com/46855061/96785092-4e196a80-13bc-11eb-93ec-74caee7baaeb.png)

**System details**
 - Home-assistant (version): 0.116.4
 - Hassio (Yes/No): No
 - alexa_media  (version from `const.py` or HA startup):  __version__ = "3.2.1"
 - alexapy (version from `pip show alexapy` or HA startup): 1.15.2
 - Amazon [2FA is enabled](https://github.com/custom-components/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). We will not debug login issues if unanswered: N

**Logs**
Please provide [logs](https://github.com/custom-components/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component). We'll be most likely asking for them anyway.
The only thing I see in the log is this:

> Logger: custom_components.alexa_media.config_flow
> Source: custom_components/alexa_media/config_flow.py:203
> Integration: Alexa Media Player (documentation, issues)
> First occurred: 4:54:48 PM (1 occurrences)
> Last logged: 4:54:48 PM
> 
> Unknown error: 'cookies_txt'

**Additional context**

**Comment by alandtse** [comment]
Duplicate of #919
